src/components/widgets/MapWidgets.tsx

```diff
@@ -24,15 +29,62 @@ const MapWidgets: React.FC = () => {
         spatialReference: view?.spatialReference
     };
 
-    const expandConfig = {
+    const basemapExpandConfig = {
         id: 'basemap-gallery-expand',
         view: view,
         content: new BasemapGallery({
             view: view,
             container: document.createElement("div"),
             id: 'basemap-gallery-widget',
         })
-    }
+    };
+
+    const searchExpandConfig = {
+        id: 'search-expand',
+        view: view,
+        content: new Search({
+            view: view,
+            container: document.createElement("div"),
+            id: 'search-widget',
+            popupEnabled: false,
+            allPlaceholder: "Find address, place, or fault zone",
```

Comment:
I'm flip-flopping I know, but please change this to just "Address, place, or fault". It's because the end of the placeholder text was getting chopped off by the right side of the search box.